Fortinet unit tests is breaking when pyFMG is not installed

<!---
Verify first that your issue/request is not already reported on GitHub.
Also test if the latest release, and master branch are affected too.
-->

##### ISSUE TYPE
<!--- Pick one below and delete the rest -->
 - Bug Report

##### COMPONENT NAME
<!---
Name of the module, plugin, task or feature
Do not include extra details here, e.g. "vyos_command" not "the network module vyos_command" or the full path
-->
fortimanager

##### ANSIBLE VERSION
<!--- Paste verbatim output from "ansible --version" between quotes below -->
```
devel branch after commit https://github.com/ansible/ansible/commit/60d2a8b6a10e083f3d550a2f657ed196a8931c28 
```

##### CONFIGURATION
<!---
If using Ansible 2.4 or above, paste the results of "ansible-config dump --only-changed"
Otherwise, mention any settings you have changed/added/removed in ansible.cfg
(or using the ANSIBLE_* environment variables).
-->

##### OS / ENVIRONMENT
<!---
Mention the OS you are running Ansible from, and the OS you are
managing, or say "N/A" for anything that is not platform-specific.
Also mention the specific version of what you are trying to control,
e.g. if this is a network bug the version of firmware on the network device.
-->

##### SUMMARY
<!--- Explain the problem briefly -->
The change https://github.com/ansible/ansible/commit/60d2a8b6a10e083f3d550a2f657ed196a8931c28 breaks unit tests when pyFMG is not installed.


##### STEPS TO REPRODUCE
<!---
For bugs, show exactly how to reproduce the problem, using a minimal test-case.
For new features, show how the feature would be used.
-->
Run unit tests without pyFMG installed.

<!--- Paste example playbooks or commands between quotes below -->
```yaml

```

<!--- You can also paste gist.github.com links for larger files -->

##### EXPECTED RESULTS
<!--- What did you expect to happen when running the steps above? -->
tests pass

##### ACTUAL RESULTS
<!--- What actually happened? If possible run with extra verbosity (-vvvv) -->

<!--- Paste verbatim command output between quotes below -->
```
________________________________________________________________________________ TestFmgrScriptModule.test_fmg_script_fail_connect ________________________________________________________________________________

self = <units.modules.network.fortimanager.test_fmgr_script.TestFmgrScriptModule testMethod=test_fmg_script_fail_connect>

    def test_fmg_script_fail_connect(self):
        set_module_args(dict(host='1.1.1.1', username='admin', password='admin', adom='root', script_name='TestScript',
                             script_type='cli', script_target='remote_device', script_description='AnsibleTest',
                             script_content='get system status'))
        result = self.execute_module(failed=True)
>       self.assertEqual(result['msg'], 'Connection to FortiManager Failed')
E       AssertionError: 'Could not import the python library pyFMG required by this module' != 'Connection to FortiManager Failed'

test/units/modules/network/fortimanager/test_fmgr_script.py:41: AssertionError
=============================================================================== 1 failed, 3167 passed, 83 skipped in 668.62 seconds ===============================================================================
```

--- a/test/units/modules/network/fortimanager/test_fmgr_script.py
+++ b/test/units/modules/network/fortimanager/test_fmgr_script.py
@@ -28,6 +28,10 @@ try:
 except ImportError:
     raise SkipTest("Could not load required modules for testing")
 
+try:
+    from pyFMG.fortimgr import FortiManager
+except ImportError:
+    raise SkipTest("FortiManager tests require pyFMG package")
 
 class TestFmgrScriptModule(TestFortimanagerModule):
 

--- a/test/units/modules/network/fortimanager/test_fmgr_script.py
+++ b/test/units/modules/network/fortimanager/test_fmgr_script.py
@@ -33,6 +33,7 @@ try:
 except ImportError:
     raise SkipTest("FortiManager tests require pyFMG package")
 
+
 class TestFmgrScriptModule(TestFortimanagerModule):
 
     module = fmgr_script

